hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -186,20 +186,25 @@
     }
     {
       name: sw_test_mode_common
-      run_opts: ["+sw_build_bin_dir={sw_build_dir}/build-bin",
+      run_opts: ["+sw_build_bin_dir={run_dir}",
                  "+sw_build_device={sw_build_device}",
-                 // Separate individual SW images supplied to +sw_images plusarg by comma.
-                 "+sw_images={eval_cmd} echo {sw_images} | sed -E 's/\\s+/,/g'"]
+                 // Format Bazel SW labels into image names separated by commas to feed into +sw_images plusarg.
```

Comment:
Could you also add an example in the comment, indicating the input and output of this `eval_cmd`?

Reply:
Done.